### ISSUE TITLE
Regions vs. Coreference

### ISSUE THREAD

**Issue by reckart** [lapps/vocabulary-pages#80]
The description of the `Region` says:

> * targets*: IDs of a sequence of annotations covering the region of primary data referred to by this annotation. Used as an alternative to start and end to point to component annotations (e.g., a token sequence) rather than directly into primary data, or to link two or more annotations (e.g., **in a coreference annotation**).

However, `Coreference` actually defines the property `mentions`  and doesn't actually use the inherited `targets` property.

**Comment by marcverhagen** [comment]
Yes, I will fix that. It is actually doubly confusing because when we link to annotations we tend to use GenericRelation and not targets. I will use another example where target are in fact used.

**Comment by marcverhagen** [comment]
Fixed this on develop branch, will keep this issue open till the fix is published in version 1.2.1 or 1.3.0.

**Comment by reckart** [comment]
Is that branch also rendered to HTML and accessible as a preview somewhere?

**Comment by marcverhagen** [comment]
It will be, but we have no automated procedure for that.

**Comment by reckart** [comment]
If you have a Jenkins - there is a plugin for making documents created as part of a build accessible via the browser. We use it our projects to make a preview of the the rendered Asciidoc documentation of current SNAPSHOT versions accessible. In case you are interested: https://wiki.jenkins.io/display/JENKINS/DocLinks+Plugin

**Comment by ksuderman** [comment]
We don't use Jenkins, but it would not be too difficult to set up Travis to build and deploy the HTML files. Everything is automated, we just have to kick it off manually with`make all upload`.

Having said that, I just did `make all upload` and the preview version is available from http://vocab.lappsgrid.org/1.3.0-SNAPSHOT
